brew/Brewfile.symlink

```diff
@@ -0,0 +1,36 @@
+tap "homebrew/bundle"
+tap "homebrew/services"
+brew "bat"
+brew "broot"
+brew "defaultbrowser"
+brew "deno"
+brew "fd"
+brew "figlet"
+brew "fish"
+brew "fzf"
+brew "gh"
+brew "git-machete"
```

Comment:
This one might be a bonus for you, but perhaps not really useful. This shows the most value when managing lots of development branches at once, and also to narrow down where an error exists in a commit.